jobconfig.toml

```diff
@@ -31,8 +33,8 @@
 [ATMOSPHERIC_TECHNICIAN]
 "Playtime Requirements" = 0
 "Required Account Age" = 0
-"Spawn Positions" = 1
-"Total Positions" = 2
+"Spawn Positions" = 3
+"Total Positions" = 3
```

Comment:
This here is fine- but in general our maps might not support multiple people spawning with the job, as well as not having the equipment etc.
I will mark occurences of this with just `.` from here on

Reply:
Reduced spawn positions to 1 in next commit just for sake of game stability, incase such causes issues. When the next commit gets ingame testing, may adjust value in a new PR to better follow 29.

Reply:
2 starting should be fine, 3 too, probably.